test/unit/org/apache/cassandra/cql3/RandomSchemaV2Test.java

```diff
@@ -98,7 +99,7 @@ public void test()
             if (mode == Mode.AccordEnabled)
             {
                 // enable accord
-                schemaChange(String.format("ALTER TABLE %s WITH " + TransactionalMode.full.asCqlParam(), metadata));
+                schemaChange(String.format("ALTER TABLE %s WITH " + TransactionalMode.full.asCqlParam() + " AND " + TransactionalMigrationFromMode.full.asCqlParam(), metadata));
```

Comment:
im not against this change but why this change?  I thought the migration mode was controlled from nodetool and not alter?  This class also doesn't do CAS (yet) so migration isn't touched?

Reply:
The transactional mode is controlled via schema, but marking ranges as migrating and finishing migration is done through `nodetool`. There is a config option to automatically mark the entire ring as migrating for a table in which case `nodetool` is only used to finish the migration for each range. Or it can just happen as a side effect of the repairs an operator might already be running.